### PR TITLE
LPS-107814 FAKE GRADLE CACHE

### PR DESCRIPTION
**Pull request by lawrence-lee**

### CODE REVIEW

File: modules/util/portal-tools-bundle-support/src/main/java/com/liferay/portal/tools/bundle/support/maven/CreateTokenMojo.java [hunk 1]
```diff
@@ -70,7 +70,7 @@ public void execute() throws MojoExecutionException {
 	)
 	protected File tokenFile;
 
-	@Parameter(defaultValue = BundleSupportConstants.DEFAULT_BUNDLE_URL)
+	@Parameter(defaultValue = BundleSupportConstants.DEFAULT_TOKEN_URL)
```
Comment (lawrence-lee) [inline modules/util/portal-tools-bundle-support/src/main/java/com/liferay/portal/tools/bundle/support/maven/CreateTokenMojo.java:73]: @david-truong, does this look like the right fix to you?

**Comment by lawrence-lee** [comment]
ci:test:gradle-plugins

File: modules/util/portal-tools-bundle-support/src/main/java/com/liferay/portal/tools/bundle/support/maven/CreateTokenMojo.java (hunk 1 above)
Comment (gamerson) [reply modules/util/portal-tools-bundle-support/src/main/java/com/liferay/portal/tools/bundle/support/maven/CreateTokenMojo.java:73]: Ah yes, the wrong constant.

**Comment by gamerson** [comment]
Pull request submitted to https://github.com/petershin/liferay-portal/pull/867. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**